### PR TITLE
Halloween: Integrate layer3

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

## Layer 3 Banner
Adds a new banner for SWAps and LIMIT ORDERS for Hallowen.

Uses Layer3 to run a marketing campaign.

## Spooky Fortunes
Uses this branch to test the spooky quotes.

Before merging we should revert the branch to be again `configuration`: https://github.com/cowprotocol/cowswap/pull/3269

# To Test
> 🚨 The banner shows only on your first load. If you navigate between pages, it disappears. I'm not sure if this is intended to be less annoying, but I think is a good thing, so I didn't pursue fixing (in case is a bug and not an issue)

Screenshot at Oct 25 18-15-48

Screenshot at Oct 25 18-16-05

Screenshot at Oct 25 18-17-26

### CODE REVIEW

**Comment by anxolin** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by anxolin** [comment]
recheck

File: apps/cowswap-frontend/index.html [hunk 1]
```diff
@@ -77,5 +77,6 @@
     </noscript>
     <div id="root"></div>
     <script type="module" src="/src/main.tsx"></script>
+    <script src="https://layer3.xyz/embed.js"></script>
```
Comment (comua) [inline apps/cowswap-frontend/index.html:80]: is there a particular reason the script is being added here instead of in the in the `<head></head>` tag?

Comment (comua) [reply apps/cowswap-frontend/index.html:80]: https://hackernoon.com/how-to-add-script-tags-in-react

Comment (anxolin) [reply apps/cowswap-frontend/index.html:80]: In the head might be blocking unless i make it deferred. I guess is possible in this case. I will experiment later to make it deferred, for now i just added using Helmet

File: apps/cowswap-frontend/index.html [hunk 2]
```diff
@@ -76,6 +76,6 @@
       </main>
     </noscript>
     <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
+    <script type="module" src="/src/main.tsx"></script>
```
Comment (alfetopito) [inline apps/cowswap-frontend/index.html:79]: Nit: unnecessary empty space :)

**Review by alfetopito — APPROVED** [review]
Looks and works ok.

Just doesn't quite work with the light mode IMO

image

**Comment by alfetopito** [comment]
And, not from this PR, but what about changing the cookie's image also in light mode? 
image

**Review by shoom3301 — APPROVED** [review]

**Comment by anxolin** [comment]
> Just doesn't quite work with the light mode IMO

You are right! @avsavsavs also brought it up.  But as for now, layer3 don't support themes. 

> And, not from this PR, but what about changing the cookie's image also in light mode?

I leave it up to @fairlighteth and Clement

**Comment by avsavsavs** [comment]
Yes, unfortunately the layer3 embeed can adapt to light mode.

In regards to all halloween in light mode, if not mistaken, Clement did have a version for the light mode, so if not a lot of effort, im more inclined to have it on both. Also logo and spider webs and all

**Pull request merged**